internal/controller/driver_controller.go

```diff
@@ -756,6 +756,32 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 								),
 							})
 						}
+						// ODF Snapshotter Sidecar Container
+						if r.driverType != NfsDriverType && snPolicy == csiv1a1.VolumeGroupSnapshotPolicy {
+							containers = append(containers, corev1.Container{
+								Name:            "odf-csi-snapshotter",
+								ImagePullPolicy: imagePullPolicy,
+								Image:           r.images["odf-snapshotter"],
+								Args: utils.DeleteZeroValues(
+									append(
+										slices.Clone(leaderElectionSettingsArg),
+										utils.LeaderElectionContainerArg,
+										utils.LogVerbosityContainerArg(logVerbosity),
+										utils.CsiAddressContainerArg,
+										utils.TimeoutContainerArg(grpcTimeout),
+										utils.ExtraCreateMetadataContainerArg,
+										utils.EnableVolumeGroupSnapshotsContainerArg,
```

Comment:
We are enabling it by default in the sidecar.

Reply:
cc @black-dragon74

Reply:
it will be enabled here not by default in the sidecar container